docs/standard/serialization/system-text-json-configure-options.md

```diff
@@ -33,12 +33,18 @@ The serializer undergoes a warm-up phase during the first serialization of each
 
 The size of the metadata cache in a `JsonSerializerOptions` instance depends on the number of types to be serialized. If you pass numerous types—for example, dynamically generated types—to the serializer, the cache size will continue to grow and can end up causing an `OutOfMemoryException`.
 
+::: zone pivot="dotnet-7-0"
+
+If the instance of `JsonSerializerOptions` that you need to use is the default instance (has all of the default settings and the default converters), use the <xref:System.Text.Json.JsonSerializerOptions.Default%2A?displayProperty=nameWithType> property rather than creating an options instance.
```

Comment:
```suggestion
If the instance of `JsonSerializerOptions` that you need to use is the default instance (has all of the default settings and the default converters), use the <xref:System.Text.Json.JsonSerializerOptions.Default?displayProperty=nameWithType> property rather than creating an options instance.
```

Reply:
Thanks!